src/__tests__/generateConfluenceMarkdown.test.ts

```diff
@@ -0,0 +1,119 @@
+import { generateConfluenceMarkdown } from '../Utilities/generateConfluenceMarkdown';
+import { Issue } from '../issueFinder';
+import { DateTime, Duration } from 'luxon';
+
+const eightDaysAgo = DateTime.utc().minus(Duration.fromISO('P8D')).toISO();
+const twoDaysAgo = DateTime.utc().minus(Duration.fromISO('P2D')).toISO();
+const fakeIssues: Issue[] = [
+    {
+        title: 'Formatting seems to use Prettier options by default',
+        createdAt: eightDaysAgo,
+        repositoryNameWithOwner: 'bc/typescript',
+        url: 'https://github.com/bc/typescript/issues/30',
+        updatedAt: '',
+        labels: ['help wanted', 'documentation'],
+    },
+
+    {
+        title: 'style config',
+        createdAt: eightDaysAgo,
+        repositoryNameWithOwner: 'material-ui/mui',
+        url: 'https://github.com/material-ui/issues/24',
+        updatedAt: '',
+        labels: ['good first issue', 'help wanted', 'documentation'],
+    },
+];
+
+const singleIssue: Issue[] = [
+    {
+        title: 'ToC: links to markdown headings',
+        createdAt: eightDaysAgo,
+        repositoryNameWithOwner: 'marmelab/react-admin',
+        url: 'https://github.com/marmelab/react-admin/issues/5620',
+        updatedAt: twoDaysAgo,
+        labels: ['good first issue', 'documentation'],
+    },
+];
+
+function roundToWholeDays(IsoString: string): number {
```

Comment:
The function is good, but the name seems a bit misleading. How about calculateAgeInWholeDays()?